README.md

```diff
@@ -23,7 +23,7 @@ This extension provides several commands in the Command Palette (F1 or Ctrl + Sh
 ```
 
 * ```checkstyle.autocheck``` - Specify whether the checkstyle extension will check Java files automatically.
-* ```checkstyle.jarPath``` - Path to the checkstyle jar file. By default, the extension will use [checkstyle-8.5-all.jar](https://sourceforge.net/projects/checkstyle/files/checkstyle/8.5/)
+* ```checkstyle.jarPath``` - Path to the checkstyle jar file. By default, the extension will use [checkstyle-8.0-all.jar](https://sourceforge.net/projects/checkstyle/files/checkstyle/8.0/)
 * ```checkstyle.configurationFile``` - Specify the checkstyle configuration file. Entering ```google_checks``` will use [google_checks.xml](https://raw.githubusercontent.com/checkstyle/checkstyle/master/src/main/resources/google_checks.xml). Entering ```sun_checks``` will use [sun_checks.xml](https://raw.githubusercontent.com/checkstyle/checkstyle/master/src/main/resources/google_checks.xml). Or you can specify the configuration file path in your local machine. By default, the extension will use ```google_checks```.
```

Comment:
why version is downgraded?

Reply:
@adashen he just follows eclipse to use a most commonly used version by default.

Reply:
Exactly as @Eskibear said.